Sources/Fuzzilli/Core/CodeTemplates.swift

```diff
@@ -139,5 +139,84 @@ public let CodeTemplates: [CodeTemplate] = [
         let arguments3 = b.generateCallArguments(for: signature)
 
         b.callMethod(propName, on: instance, withArgs: arguments3)
-    }
+    },
+    CodeTemplate("JIT2Functions") { b in
+        let genSize = 3
+
+        // Generate random function signatures as our helpers
+        var functionSignatures = CodeTemplate.generateRandomFunctionSignatures(forFuzzer: b.fuzzer, n: 2)
+
+        // Generate random property types
+        CodeTemplate.generateRandomPropertyTypes(forBuilder: b)
+
+        // Generate random method types
+        CodeTemplate.generateRandomMethodTypes(forBuilder: b, n: 2)
+
+        b.generate(n: genSize)
+
+        // Generate some small functions
+        for signature in functionSignatures {
+            // Here generate a random function type, e.g. arrow/generator etc
+            b.definePlainFunction(withSignature: signature) { args in
+                b.generate(n: genSize)
+            }
+        }
+
+        // Generate a larger function
+        let signature1 = CodeTemplate.generateSignature(forFuzzer: b.fuzzer, n: 4)
+        let f1 = b.definePlainFunction(withSignature: signature1) { args in
+            // Generate (larger) function body
+            b.generate(n: 15)
+        }
+
+        // Generate a second larger function
+        let signature2 = CodeTemplate.generateSignature(forFuzzer: b.fuzzer, n: 4)
+        let f2 = b.definePlainFunction(withSignature: signature2) { args in
+            // Generate (larger) function body
+            b.generate(n: 15)
+        }
+
+        // Generate some random instructions now
+        b.generate(n: genSize)
+
+        // trigger JIT for first function
+        b.forLoop(b.loadInt(0), .lessThan, b.loadInt(100), .Add, b.loadInt(1)) { args in
+            let arguments = b.generateCallArguments(for: signature1)
+            b.callFunction(f1, withArgs: arguments)
+        }
+
+        // trigger JIT for second function
+        b.forLoop(b.loadInt(0), .lessThan, b.loadInt(100), .Add, b.loadInt(1)) { args in
+            let arguments = b.generateCallArguments(for: signature2)
+            b.callFunction(f2, withArgs: arguments)
+        }
+
+        // more random instructions
+        b.generate(n: genSize)
+
+        let argumentsBeforeJit2 = b.generateCallArguments(for: signature2)
+        b.callFunction(f2, withArgs: argumentsBeforeJit2)
```

Comment:
Nit: maybe you can just make it `b.callFunction(f2, withArgs: b.generateCallArguments(for: signature2))` here and below instead of creating the temporary variables?